tsconfig.json

```diff
@@ -22,6 +22,6 @@
         "strict": true,
         "target": "es5"
     },
-    "exclude": ["node_modules"],
+    "exclude": ["node_modules", "src/index.ts"],
```

Comment:
This prevents VS Code from suggesting imports from the index file, which is meant for consumer use only.